packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { DocsContext } from '@storybook/addon-docs';
+import dedent from 'dedent';
+import Box from '~components/Box';
+
+export type SandboxProps = {
+  children: string;
+  language?: 'js' | 'ts' | 'jsx' | 'tsx';
+};
+
+function Sandbox({ children, language = 'tsx' }: SandboxProps): JSX.Element {
+  const {
+    // @ts-expect-error globals is available but the typings in storybook are properly defined hence, ignoring it
+    globals: { themeTokenName, colorScheme },
+  } = React.useContext(DocsContext);
+
+  return (
+    <Box>
+      <br />
+      <Sandpack
+        template="react-ts"
+        files={{
+          '/index.tsx': dedent`import { StrictMode } from "react";
+            import * as ReactDOMClient from "react-dom/client";
+            import styled, { createGlobalStyle } from "styled-components";
+  
+            import { BladeProvider, Theme } from "@razorpay/blade/components";
+            import { ${themeTokenName} } from "@razorpay/blade/tokens";
+            import "@fontsource/lato/400.css";
+            import "@fontsource/lato/700.css";
+            
+            import App from "./App";
+  
+            const BackgroundBox = styled.div(
+              ({ theme }: { theme: Theme }) => ({
+                backgroundColor: theme.colors.surface.background.level1.lowContrast,
+                height: '100vh',
+                width: '100vw',
+                padding: '12px'
+              })
+            );
+
+            const GlobalStyles = createGlobalStyle\`
+              body {
+                margin: 0;
+                padding: 0;
+              }
+            \`;
+            
+            const rootElement = document.getElementById("root");
```

Comment:
does sandpack creates this root or storybook?

Reply:
Sandpack does. Inside an iframe.